libs/commons/src/index.ts

```diff
@@ -0,0 +1,4 @@
+// export * from './lib/constants';
+// export * from './lib/legos';
+// export * from './lib/types';
+// export * from './lib/utils';
```

Comment:
This should be removed, probably along with the empty files

Reply:
You mind if I add one util/example file for each? I've got to bring them back right away.

Reply:
Actually I just removed all the empty things. GTG!